Cargo.toml

```diff
@@ -72,16 +65,15 @@ dyn-clone = "1.0"
 uuid = { version = "1", features = ["serde", "v4"] }
 derive_more = "0.99"
 paste = "1.0"
-libc = "0.2"
 sata = { git = "https://github.com/Satellite-im/Sata", rev = "9b5a5d441c816968bc75ea8b99335f728be259ef" }
 tracing = { version = "0.1" }
-tracing-futures = { version = "0.2" }
-getrandom = { version = "0.2" }
 either = "1"
 void = "1"
 
-# ipfs dependency
-rust-ipfs = "0.11.0"
+#ipfs dependency
+rust-ipfs = { version = "0.11.0", default-features = false, features = [
+    "libp2p_bitswap",
+] }
```

Comment:
```suggestion
rust-ipfs = "0.11.0"
```

Reply:
why? we only need the specific feature, no?

Reply:
The feature isnt needed here anymore (the default-features was not needed either).

Reply:
ok but why enable all features then?

Reply:
It doesnt contain any default features.

Reply:
ah ok. :) That's the detail I missed.